src/components/LazyLog/index.jsx

```diff
@@ -177,6 +181,7 @@ export default class LazyLog extends Component {
     loadingComponent: Loading,
     lineClassName: '',
     highlightLineClassName: '',
+    caseInsensitive: false,
```

Comment:
Just curious, why is `caseInsensitive` not `true` by default?

Reply:
It was not available before, so kept it as false by default.
So if its needed it can be enabled.

Reply:
It would probably be a breaking change if we make it `true` by default.